### ISSUE TITLE
aircrack-ng - huge memory consumption (leak?) with hex wep dictionary

### ISSUE THREAD

**Issue by jbenden** [jbenden/aircrack-ng-migration#1138]
**Reported by varchar on 7 Jun 2013 09:57 UTC**

Aircrack-ng 1.2 beta1 with following command:
```
aircrack-ng -a 1 -n 64 -w h:wep64hex some.cap 
```
and with content of wep64hex dict like this: 
```
...
06:37:80:25:61
06:37:80:25:66
06:37:80:27:16
06:37:80:27:19
06:37:80:27:61 
...
```
eat GiB's of RAM after few minutes of run, and then must/will be killed.
----
Here not the right place to ask, but how to use stdin as hex dict?
``` -w h:- ```
does not work for me, unlike
``` -w - ```
Thanx and sorry.

Migrated-From: https://trac.aircrack-ng.org/ticket/1138

**Comment by jbenden** [comment]
**Comment by varchar on 7 Jun 2013 09:57 UTC**

Found this as duplicate of [#685](http://trac.aircrack-ng.org/ticket/685)

**Comment by jbenden** [comment]
**Modified by misterx on 7 Jun 2013 09:57 UTC**

**Comment by jbenden** [comment]
**Comment by anonymous on 7 Jun 2013 09:57 UTC**

It will be fixed or no? 685 (with patch attached) 4 years old!

**Comment by jbenden** [comment]
**Comment by anonymous on 7 Jun 2013 09:57 UTC**

Have you tested the patch? It doesn't fix the issue.

**Comment by jbenden** [comment]
**Comment by misterx on 7 Jun 2013 09:57 UTC**

Fixed in aircrack-ng/aircrack-ng@a4233bbd8d9b8e582ce229cdf68c4c40ec20bc31.
